devtools/dump_devinfo.py

```diff
@@ -64,6 +76,8 @@ def scrub(res):
                     v = base64.b64encode(b"#MASKED_NAME#").decode()
                 elif k in ["alias"]:
                     v = "#MASKED_NAME#"
+                elif isinstance(res[k], int):
+                    v = 0
```

Comment:
Setting to zero is consistent with 
```
else:
    v = re.sub(r"\w", "0", v)
```
and
```
  if k in ["latitude", "latitude_i", "longitude", "longitude_i"]:
      v = 0
```
This int check was added because the AP information returns a wifi channel as an int.  As there is no channel 0 on wifi I think this should be recognised as scrubbed.